Meeqan/buys
# #163876265 Added JWT authentication

**What does this PR do?**

This PR implements the JWT authentication endpoints.

**Description of Task to be completed?**
As a developer, I should be able to generate a JWT token making a `POST` request with the valid details to `/user/token`.


**How should this be manually tested?**
- Pull the branch `git fetch origin ft-add-jwt-authentication-163876265`
- Check into it` git checkout ft-add-jwt-authentication-163876265`
- Activate the virtualenv
- Run the app: python app/manage.py runserver
- On postman make a POST request to `{{base-url}}/api/v1/user/token`

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#163876265](https://www.pivotaltracker.com/n/projects/2170023/stories/163876265)
